### ISSUE TITLE
Linux Support?

### ISSUE THREAD

**Issue by maktoobgar** [RedMser/godot-embed-external-editor#12]
## Linux Support Maybe in the Future?

Is this addon possible to be used in Linux? (many godot developers, like me, are using Linux)
Please if it is possible, consider supporting it.

**Comment by Shadowblitz16** [comment]
PLEASE!

**Comment by RedMser** [comment]
I don't believe there exists a similar API on Linux to change a window's parent without doing "always on top". Multiple different window managers makes this harder as well.

PRs for this are welcome, but I'm using Windows so that's where my personal focus will be for now. Hope you understand!
